fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -1594,7 +1590,7 @@ export class PseudoFWC {
         this.failuresRight.push(...failureKeysRight);
 
         if (tempFailureArrayLeft.length > 0) {
-            this.memoMessageLeft.set(orderedFailureArrayLeft);
+            this.ewdMessageLinesLeft.forEach((l, i) => l.set(orderedFailureArrayLeft[i]));
```

Comment:
Doesn't it need to set them to zero also if the tempFailureArrayLeft has nothing in it?

Reply:
I think that's being dealt with in https://github.com/flybywiresim/a32nx/pull/7921/files/7c80f4c7011d3ac61914ff47d97fc2fd3a798c7c#diff-2439d534e1bce3f0a530cd3d6eefdf08aae38febfb1f9300a2ae75afed4661d5R1633 and following. At least I think the logic of this  isn't touched by my changes.